packages/colony-contract-loader-network/src/NetworkLoader.js

```diff
@@ -0,0 +1,49 @@
+import assert from "assert";
+import ContractLoader, { truffleTransform } from "@colony/colony-js-contract-loader";
+
+import config from "../config.json";
+
+const NETWORKS = ["rinkeby"];
+const LATEST_VERSION = config.LATEST_CONTRACT_VERSION;
+const VERSIONED_CONTRACTS = ["IColony", "IColonyNetwork", "Authority"];
+const STATIC_CONTRACTS = ["EtherRouter", "Token"];
+
+const CONTRACTS = {};
+NETWORKS.forEach(network => {
+  [...Array(LATEST_VERSION).keys()].forEach(version => {
```

Comment:
This seems a little hard to understand. `LATEST_VERSION` is the string `1`, but `version` in this loop is `0` (and we add one to that to define a version for each network). 

Won't `LATEST_VERSION` always be just one entry? And aren't we ignoring it here?